wgpu-core/src/id.rs

```diff
@@ -1,9 +1,27 @@
 use crate::{Epoch, Index};
-use std::{cmp::Ordering, fmt, marker::PhantomData, num::NonZeroU64};
+#[cfg(feature = "id32")]
+use std::num::NonZeroU32;
+#[cfg(not(feature = "id32"))]
+use std::num::NonZeroU64;
```

Comment:
let's import it as `NonZeroIndex` in both cases, to avoid branching where we are using it